blocks/header/header.css

```diff
@@ -324,3 +324,309 @@ header .breadcrumbs ol li[aria-current] {
+  --spectrum-switch-border-color-selected-hover: var(--spectrum-gray-900);
+  --spectrum-switch-border-color-selected-down: var(--spectrum-gray-900);
+  --spectrum-switch-border-color-selected-focus: var(--spectrum-gray-900);
+
+  --spectrum-switch-background-color: var(--spectrum-gray-200);
+  --spectrum-switch-background-color-disabled: var(--spectrum-gray-25);
+
+  --spectrum-switch-background-color-selected-default: var(--spectrum-blue-900);
+  --spectrum-switch-background-color-selected-hover: var(--spectrum-blue-900);
+  --spectrum-switch-background-color-selected-down: var(--spectrum-blue-900);
+  --spectrum-switch-background-color-selected-focus: var(--spectrum-blue-900);
+  --spectrum-switch-background-color-selected-disabled: var(
+    --spectrum-gray-400
+  );
+
+  --spectrum-switch-focus-indicator-thickness: var(
+    --spectrum-focus-indicator-thickness
+  );
+  --spectrum-switch-focus-indicator-color: var(
+    --spectrum-focus-indicator-color
+  );
+  --spectrum-switch-focus-indicator-gap: var(--spectrum-focus-indicator-gap);
+
+  --spectrum-switch-handle-background-color-default: var(--spectrum-gray-50);
+  --spectrum-switch-handle-background-color-hover: var(--spectrum-gray-50);
+  --spectrum-switch-handle-background-color-down: var(--spectrum-gray-50);
+  --spectrum-switch-handle-background-color-focus: var(--spectrum-gray-50);
+  --spectrum-switch-handle-background-color-disabled: var(--spectrum-gray-50);
+  --switch-handle-border-color: var(--spectrum-gray-600);
+
+  --spectrum-switch-handle-background-color-selected: var(--spectrum-gray-50);
+  --spectrum-switch-handle-background-color-selected-disabled: var(
+    --spectrum-gray-25
+  );
+
+  --switch-handle-border-color--selected: var(--spectrum-blue-900);
+
+  --spectrum-switch-handle-size: 1.125rem;
+  --spectrum-switch-handle-selected-size: var(--spectrum-switch-handle-size);
+
+  --spectrum-switch-min-height: 2rem;
+  --spectrum-switch-control-width: 2rem;
+  --spectrum-switch-control-height: 1.125rem;
+  --spectrum-switch-control-label-spacing: 1rem;
+  --spectrum-switch-spacing-top-to-control: 2rem;
+  --spectrum-switch-spacing-top-to-label: 1rem;
+  --spectrum-switch-spacing-bottom-to-label: 1rem;
+
+  --spectrum-switch-border-radius: var(--spectrum-corner-radius-1000);
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 1rem;
+  position: relative;
+  min-block-size: var(--spectrum-switch-min-height);
+  max-inline-size: 100%;
+  vertical-align: top;
+}
+
+.theme-toggle__input {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+
+  position: absolute;
+  inline-size: 100%;
+  block-size: 100%;
+  inset-block-start: 0;
+  inset-inline-start: 0;
+  opacity: 0;
+  z-index: 1;
+
+  cursor: pointer;
+
+  &:disabled,
+  &[disabled] {
+    cursor: default;
+  }
+}
+
+.theme-toggle__label {
+  margin-right: 0.25rem;
+  transition: color 200ms ease-in-out;
+  color: var(--color-text);
+
+  .theme-toggle:hover ~ & {
```

Comment:
Lines 441, 445, and 449 need to be descendant selectors instead of sibling selectors.